util/dvsim/utils.py

```diff
@@ -343,3 +343,82 @@ def color_cell(cell, cclass, indicator="%"):
         for item in subst_list:
             text = text.replace(item, subst_list[item])
     return text
+
+
+def gen_css_style(use_css_style_for_web):
```

Comment:
IMO this will never be used as a standalone routine. It can be nested under `ms_results_to_html`.

Reply:
Oh I see, thanks for the suggestion, I will move it into `ms_results_to_html`.